frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -188,13 +198,14 @@ public void refreshModel() {
 
       refreshModel(macheteFilePath, isMacheteFilePresent);
     } else {
-      LOG.warn("No git repository selected, not updating the model");
+      LOG.warn("Selected git repository is undefined; unable to refresh model");
     }
   }
 
   @UIEffect
-  private void setTextForEmptyGraph(String upperText, String lowerText) {
-    getEmptyText().setText(upperText).appendSecondaryText(lowerText, StatusText.DEFAULT_ATTRIBUTES, /* listener */ null);
+  private void setTextForEmptyTable(String upperText, String lowerText, @UI Runnable onClickRunnableAction) {
+    var attrs = new SimpleTextAttributes(SimpleTextAttributes.STYLE_PLAIN, JBUI.CurrentTheme.Link.linkColor());
+    getEmptyText().setText(upperText).appendSecondaryText(lowerText, attrs, /* listener */ e -> onClickRunnableAction.run());
```

Comment:
Nit: maybe `s/e/__` to (un-)highlight an ignored param

Reply:
Btw single underscore `_` is reserved to mean ignored lambda param in some future version of Java